src/core/properties/directoryproperty.cpp

```diff
@@ -53,4 +54,35 @@ std::string DirectoryProperty::getClassIdentifierForWidget() const {
     return FileProperty::getClassIdentifier();
 }
 
+void DirectoryProperty::requestFile() {
+    for (auto widget : getWidgets()) {
+        if (auto filerequestable = dynamic_cast<FileRequestable*>(widget)) {
+            if (filerequestable->requestFile()) return;
+        }
+    }
+    if (getWidgets().empty()) {
+
+        const std::string filename{ this->get() };
+
+        // Setup Extensions
+        std::vector<FileExtension> filters = this->getNameFilters();
+
+        auto fileDialog = util::dynamic_unique_ptr_cast<FileDialog>(
+            InviwoApplication::getPtr()->getDialogFactory()->create("FileDialog"));
+        if (!fileDialog) {
+            return;
+        }
+        fileDialog->setTitle("Open Folder");
```

Comment:
These should be set based on the FileProperty

Reply:
Like this fc0bade?

Reply:
No, like in https://github.com/inviwo/inviwo-dev/blob/4fd6dfe2e7df8e05e0fd4b8ce5400a051e33024e/modules/qtwidgets/properties/filepropertywidgetqt.cpp

Line 133 and onwards